js_modules/dagit/packages/core/src/asset-graph/AssetNode.tsx

```diff
@@ -194,8 +194,9 @@ export function buildAssetNodeStatusRow({
   }
 
   if (liveData.partitionStats) {
-    const {numPartitions, numMaterialized} = liveData.partitionStats;
-    const numMissing = numPartitions - numMaterialized;
+    const {numPartitions, numMaterialized, numFailed} = liveData.partitionStats;
+    const numMissing = numPartitions - numMaterialized - numFailed;
```

Comment:
I think I've heard there's a "Failed Materialized" state? Or was that "Success, not materialized" ?

Reply:
It's a bit interesting -- a single partition key can be materialized, and then a backfill of it can fail, so the same key can be in both states. We decided to implement these numbers on the backend so that each key can only be in one bucket though, and failed trumps materailized. 

I think it might be worth a call-out in this code though 👍